c7n_mailer lambda error after included slack integration

Hi team,

It appears that deployed lambda mailer is having some problem with the libs, after the last commit that included slack integration. 

After update c7n_mailer lambda I'm seeing this error in the logs:
```
[ERROR] 2018-05-01T00:45:26.698Z f04d1326-4cd8-11e8-9f05-670c91cd10c1 Error starting mailer MailerSqsQueueProcessor().
Error: No module named requests

Traceback (most recent call last):
File "/var/task/c7n_mailer/handle.py", line 46, in start_c7n_mailer
mailer_sqs_queue_processor.run(parallel)
File "/var/task/c7n_mailer/sqs_queue_processor.py", line 126, in run
self.process_sqs_message(sqs_message)
File "/var/task/c7n_mailer/sqs_queue_processor.py", line 170, in process_sqs_message
from .slack_delivery import SlackDelivery
File "/var/task/c7n_mailer/slack_delivery.py", line 17, in <module>
import requests
ImportError: No module named requests
```
 I've had no success even with a fresh install from the repos.

--- a/tools/c7n_mailer/c7n_mailer/deploy.py
+++ b/tools/c7n_mailer/c7n_mailer/deploy.py
@@ -41,7 +41,7 @@ def dispatch(event, context):
 def get_archive(config):
     archive = PythonPackageArchive(
         'c7n_mailer', 'ldap3', 'pyasn1', 'jinja2', 'markupsafe', 'ruamel',
-        'redis')
+        'redis', 'datadog', 'slackclient', 'requests')
 
     template_dir = os.path.abspath(
         os.path.join(os.path.dirname(__file__), '..', 'msg-templates'))

--- a/tools/c7n_mailer/setup.py
+++ b/tools/c7n_mailer/setup.py
@@ -16,7 +16,7 @@ from __future__ import absolute_import, division, print_function, unicode_litera
 from setuptools import setup, find_packages
 import os
 
-requires = ["Jinja2", "boto3", "jsonschema", "ruamel.yaml<0.15", "datadog"]
+requires = ["Jinja2", "boto3", "jsonschema", "ruamel.yaml<0.15", "datadog", "slackclient"]
 try:
     from concurrent import futures
 except ImportError:

--- a/tools/c7n_mailer/setup.py
+++ b/tools/c7n_mailer/setup.py
@@ -16,6 +16,7 @@ from __future__ import absolute_import, division, print_function, unicode_litera
 from setuptools import setup, find_packages
 import os
 
+# *Any* updates here should also go into c7n_mailer/deploy.py for lambda packaging.
 requires = ["Jinja2", "boto3", "jsonschema", "ruamel.yaml<0.15", "datadog", "slackclient"]
 try:
     from concurrent import futures

--- a/tools/c7n_mailer/setup.py
+++ b/tools/c7n_mailer/setup.py
@@ -17,7 +17,8 @@ from setuptools import setup, find_packages
 import os
 
 # *Any* updates here should also go into c7n_mailer/deploy.py for lambda packaging.
-requires = ["Jinja2", "boto3", "jsonschema", "ruamel.yaml<0.15", "datadog", "slackclient"]
+requires = [
+    "Jinja2", "boto3", "jsonschema", "ruamel.yaml<0.15", "datadog", "slackclient", "ldap3"]
 try:
     from concurrent import futures
 except ImportError:

